conanfile.py

```diff
@@ -197,9 +224,35 @@ def package_info(self):
             "include/OGRE/RTShaderSystem",
             "include/OGRE/Terrain",
             "include/OGRE/Volume",
+            "include/OGRE/Threading",
+            "include/OGRE/MeshLodGenerator",
+            "include/OGRE/Plugins/BSPSceneManager",
+            "include/OGRE/Plugins/CgProgramManager",
+            "include/OGRE/Plugins/DotScene",
+            "include/OGRE/Plugins/FreeImageCodec",
+            "include/OGRE/Plugins/OctreeSceneManager",
+            "include/OGRE/Plugins/OctreeZone",
+            "include/OGRE/Plugins/ParticleFX",
+            "include/OGRE/Plugins/PCZSceneManager",
+            "include/OGRE/Plugins/STBICodec"
         ])
         if self.options.bites:
             self.cpp_info.includedirs.append("include/OGRE/Bites")
+            
+        if self.options.opengl_renderer:
+            self.cpp_info.libs.append("include/OGRE/RenderSystems/GL")
```

Comment:
Did you mean to add to `self.cpp_info.includedirs` and not `libs`. Same goes for entries below